frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -343,13 +342,12 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
       .then((storageClasses: StorageClassResourceKind[]) => {
         // find all storageclass with the given provisioner
         const scList = _.filter(storageClasses, (sc) => sc.provisioner === provisioner);
-        // take the default storageclass
-        _.forEach(scList, (sc) => {
-          if (isDefaultClass(sc)) {
-            storageClass = sc.metadata.name;
-          }
-        });
-        makeOCSRequest();
+        // take the first provisioner based storageclass
+        const firstSC = _.get(scList, '0');
```

Comment:
Does the 0th index of the list will always be picked up or it will be picked when default storage class is not present

Reply:
Correct if sclist is having the default storage class then we should use it otherwise the index 0.